woltka/classify.py

```diff
@@ -135,6 +135,36 @@ def count(matches):
     return res
 
 
+def count_strata(matches, strata):
+    """Stratify taxa in a map and count occurrences.
+
+    Parameters
+    ----------
+    matches : dict of str or dict
+        Query-to-taxon(a) map.
```

Comment:
Not sure what `taxon(a)` means here

Reply:
It means "taxon or taxa", analogous to "subject(s)".